ansible.config.yml.template

```diff
@@ -0,0 +1,82 @@
+---
+use_local_repository: 'no'
+## remote_repository_url is an object that contains two git urls of the project.
+## If the git url(s) is/are changed, please uncomment the object below and fill in the new url(s).
+# remote_repository_url:
+#   jiskefet_api: https://github.com/SoftwareForScience/jiskefet-api.git
+#   jiskefet_ui: https://github.com/SoftwareForScience/jiskefet-ui.git
+# repository_branch:
+#   jiskefet_api: master
+#   jiskefet_ui: master
+
+## USE_CERN_SSO is set to 'false' by default, please uncomment the line below if you wish to use CERN's SSO provider
+## instead of Github's SSO.
+# USE_CERN_SSO: 'false'
+
+jiskefet_api_general_settings:
+  ## USE_API_BASE_PATH is set to 'true' by default, please uncomment the line below if you wish to use the API without the 
+  ## '/api' base path. Please note when setting the USE_API_BASE_PATH value to 'false', 
+  ## it will cause the UI to fail when sending requests to the API.
+  # USE_API_BASE_PATH: 'true'
```

Comment:
Shouldn't this value be 'false' if the default is true?

Reply:
It does not matter what the value is, because I want to remove all the comments from the template file and refer to the options [here](https://github.com/SoftwareForScience/jiskefet-deploy/blob/improvement/O2B-44-deploy-optimalization/docs/configuration_file.md#list-of-extra-configuration-fields).

Reply:
Sounds good.